django/utils/autoreload.py

```diff
@@ -16,7 +16,7 @@
 
```

Comment:
> the development experience is greatly impacted if the autoreloader does not function correctly. The failure mode here is if the request_finished signal is not correctly dispatched, and if this ever happens for any reason then the autoreloader will fail outright. There is historical precedent for this and while I'm not aware of anything else like this it does feel like it might crop up somehow during development - perhaps in ways that have so far gone unnoticed.


Wouldn't it be better to have the autoreloader fail due to the signal not being dispatched and catch the bug quickly than to risk someone's code run into issues in a prod site or something where they are depending on the signal for some process. (I am not aware of anyone using this on prod, just a curious thought.)